docs/fastdds/dds_layer/core/status/status.rst

```diff
@@ -430,33 +432,34 @@ List of status data members:
   Handle to the last instance whose sample was rejected.
   If no sample was ever rejected, it will have value ``c_InstanceHandle_Unknown``.
 
-.. warning::
-
-    Currently this status is not supported and will be implemented in future releases.
-    As a result, trying to access this status will return ``NOT_SUPPORTED``
-    and the corresponding listener will never be called.
-
-
 .. _dds_layer_core_status_sampleRejectedStatusKind:
 
 SampleRejectedStatusKind
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-There are four possible values (see |SampleRejectedStatusKind-api|):
-
-* |NOT_REJECTED|:
-  It means there have been no rejections so far on this DataReader.
-  The rejection reason will have this value only while the total count of rejections is zero.
-* |REJECTED_BY_INSTANCES_LIMIT|:
-  The sample was rejected because the
-  :ref:`max_instances<resourcelimitsqospolicy>` limit was reached.
-* |REJECTED_BY_SAMPLES_LIMIT|:
-  The sample was rejected because the
-  :ref:`max_samples<resourcelimitsqospolicy>` limit was reached.
-* |REJECTED_BY_SAMPLES_PER_INSTANCE_LIMIT|:
-  The sample was rejected because the
-  :ref:`max_samples_per_instance<resourcelimitsqospolicy>` limit was reached.
+Samples are rejected due to resource limits.
+But this doesn't mean they are lost.
+A rejected sample may be accepted in the future.
```

Comment:
In *Fast DDS*, samples can be rejected due to resource limit reasons. 
However, the fact that the samples are rejected does not imply that they are lost, i.e. a rejected sample may be accepted in the future.

Reply:
Fixes in f97ad9c